source/plugins/go/src/utils.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/Azure/azure-kusto-go/kusto"
 	"github.com/Azure/azure-kusto-go/kusto/ingest"
 	"github.com/tinylib/msgp/msgp"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
```

Comment:
how  tracking the error message with no error in windows fluent-bit?

Reply:
sorry I didnt understand what you are trying to say

Reply:
> sorry I didnt understand what you are trying to say

lets discuss about this along with other issues cpu and vuln fixes in tomorrow standup. lets go-ahead and merge this change since its blocking.